hledger/Hledger/Cli/Commands/Balance.hs

```diff
@@ -291,6 +291,8 @@ balancemode = hledgerCommandMode
       ])    
     ,flagNone ["valuechange"] (setboolopt "valuechange")
       "show change of value of period-end historical balances"
+    ,flagNone ["gain"] (setboolopt "gain")
+      "show gain (present valuation minus cost basis) of period-end historical balances"
```

Comment:
I think I missed this one; do you think this is better ?

show unrealised capital gain/loss (period-end historical balance value minus each amount's original cost)

Reply:
Done.